modules/rosapps/readme.txt

```diff
@@ -2,23 +2,18 @@
 ReactOS Applications
 ====================
 
-This directory contains extra applicatons for ReactOS.
+This directory contains extra applications for ReactOS.
 Make sure you also have a copy of the rest of the ReactOS
 source before you attempt to build anything in this module.
 It is to be placed under "modules" subdirectory of a trunk checkout.
-No additional changes to build files are necessary, it'll be picked up
-automatically.
+Module require to be enabled during "configure" process.
 
-To include "rosapps" in your build folder:
-1. Copy the rosapps folder into the reactos\modules folder, or
-2. Link reactos/modules/rosapps to rosapps
+To include module in your build folder, run configure script with flags -DENABLE_ROSAPPS=1 
```

Comment:
To include **the** module..., run **the** configure script with **the** flags ...